bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/dto/Status.java

```diff
@@ -63,31 +63,31 @@
     public String serviceWarningStatus = "";
     private @NonNullByDefault({}) List<Object> bulbFailures;
 
-    public Optional<WindowsStatus> getWindows() {
+    public @NonNullByDefault({}) Optional<WindowsStatus> getWindows() {
```

Comment:
Why is `@NonNullByDefault({})` needed here ?

Reply:
Eclipse is picking up the NonNullByDefault and applying it to the `Optional.ofNullable`, which then means it flags as an error. The right way to fix would be external annotations, but they don't seem to be applied by default in openhab projects, so for developer convenience I thought just adding these would be safe and easy. It means the project will open for everybody without errors regardless of if they have the external annotations sorted.

Reply:
Happy to remove them from the PR if you prefer. I know there is some work in openhab to get the external annotations to work by default, but it didn't work for me :(

And, if it didn't work for me, I can imagine other developers hitting the same problem, so it lowers the barrier to entry for devs. Any hints on making that work would also be appreciated by me, I might even consider updating the developer doc to help others use it.

Reply:
Yes, I prefer, my eclipse IDE does not complain on his side.